app/jobs/benchmark_regression_job.rb

```diff
@@ -0,0 +1,16 @@
+class BenchmarkRegressionJob < ActiveJob::Base
+  queue_as :default
+ 
+  def perform(initiator, benchmark_type, benchmark_result_type, category)
+    commit_objects = BenchmarkRun.where( initiator: initiator, benchmark_type: benchmark_type,
+                     benchmark_result_type: benchmark_result_type).limit(10)
```

Comment:
I added a post in http://community.rubybench.org/t/gsoc-project-improving-rubybench/99/55 to discuss the method.